indexer-app/resources/schemas/dataqualitysummary/v1.0.0/index.json

```diff
@@ -12,6 +12,12 @@
         "Field": ".Name",
         "Name": "Name",
         "Mapping": "string"
+      },
+      {
+        "Description": "Id - This is the old ECHO 10 GUID.",
+        "Field": ".Id",
+        "Name": "Id",
+        "Mapping": "string"
       }
     ]
   }
```

Comment:
Is this the file that defines all the dataqualitysummary metadata fields that will be indexed?  If so, how do we add the association fields to it?

Reply:
I see you also have indexer-app/src/cmr/indexer/data/concepts/generic.clj, which we can add association index to...

Reply:
yes, it is.  I will be adding those in the coming days. There is no difference, they are the same. Initially we wanted 1 file to hold data for ingest and indexer, so we put these together.  It might make sense to break these apart if ingest doesn't need it.